ngambino0192/stocks
# Styles auth merge

Opening this up because I think we should probably just work off of this branch and stick to a single docker config. I deviated from it mainly because I was having trouble accessing the db in the previous setup. Hopefully this will be okay for our needs going forward.

To test auth:

1. Open up postman
2. Create a POST request with the following:
- `http://localhost:6969/api/user/create`
- raw JSON in body:
```
{
    "username": "toby",
    "email": "toby@test.com",
    "password": "password"
}
```

3. Create another POST request:
- `http://localhost:6969/api/user/login`
- raw JSON in body:
```
{
    "email": "toby@test.com",
    "password": "password"
}
```

I will be cleaning this up and breaking out the routes in a better pattern in a future PR (probably tomorrow). I'm also hoping to get password reset logic going in a separate task.

Also not sure if I missed any of your styles. Could you give it a sanity check?
